annotators/user_knowledge_memorizer/server.py

```diff
@@ -544,9 +627,16 @@ def memorize(graph, uttrs):
         triplets_already_in_kg_batch.append(triplets_already_in_kg + properties_already_in_kg)
 
     logger.info(
-        f"added_to_graph -- {triplets_added_to_kg_batch}, triplets_already_in_graph -- {triplets_already_in_kg_batch}"
+        f"added_to_graph -- {triplets_added_to_kg_batch}, "
+        f"triplets_already_in_graph -- {triplets_already_in_kg_batch}, kg_prompt -- {related_knowledge}"
```

Comment:
related_knowledge might be unbound and consequently raise an error. So, add `related_knowledge = []` before `for utt in uttrs:`

Reply:
Rami, didn't get your idea here.
`related_knowledge` should not be unbound as I provide a default value at line 571.